### ISSUE TITLE
Reverse address search issues 

### ISSUE THREAD

**Issue by e5575177** [osm-search/Nominatim#1210]
Hi,
I've noticed some inconsistencies when using reverse address search.

When I try to reverse search an address for 43.399 24.625, the address seems correct but it also shows two digferent districs - Mara Dencheva (Мара Денчева) and Storgoziya (Сторгозия) along with Storgozia's postal code. The correct one is Mara Dencheva. Storgozia is shown in different parts of the city.

When I search (forward search) for Pleven Storgoziya - the district is marked correctly.

I am not sure where the problem resides, but I think there is a problem how Storgoziya district is defined.

It will be much appreciated if the issue can be resolved.

**Comment by e5575177** [comment]
Hi,

Most probably the coordinates, which I gave you were not exactly correct, sorry for that.

Now I've tried with the following:

43.3979696,24.6258210681193

Which returns:


бл.24, Manush Voyvoda, кв. Мара Денчева, ж.к. Сторгозия, Pleven, 5805, Bulgaria (Building)

Basically Сторгозия should be only defined as residential (in different part of the city). I think that Сторгозия should be defined only as a residential, and not as suburb (as it is here: https://nominatim.openstreetmap.org/details.php?place_id=103622039)

Basically - there should be no suburbs defined in Pleven.

P.S. I am on mobile right now, but I hope that I've managed to describe the issue properly.

**Comment by mtmail** [comment]
(1) and (2) sound good. In my opinion `landuse=residential` features shouldn't have a names. And ideally https://www.openstreetmap.org/node/528962528, if it's really a suburb, should be replaced by a relation (likely as big or slightly bigger as the landuse relation) because with nodes Nominatim has to guess which suburb is closest to a search position.

> there should be no suburbs defined in Pleven

Fine. I mean you're probably more experienced with Bulgarian addresses and administrative hierarchy than us.

**Comment by e5575177** [comment]
Ok, so can I assist somehow with these changes?

**Comment by mtmail** [comment]
@e5575177 Absolutely, you can do all the changes or leave a map note for local mappers (https://www.openstreetmap.org/fixthemap). I need to close the discussion here because the search logic (Nominatim) itself seems fine, just the underlying OSM data needs changes.

**Comment by mtmail** [comment]
@e5575177 Absolutely, you can do all the changes or leave a map note for local mappers (https://www.openstreetmap.org/fixthemap). I need to close the discussion here because the search logic (Nominatim) itself seems fine, just the underlying OSM data needs changes.
